installing_on_kubernetes/_topics/prepare_kubernetes_namespace.md

```diff
@@ -27,7 +27,9 @@ The operator directory referenced below can be found [here]({{ site.data.product
    $ oc create -f config/rbac/service_account.yaml
    ```
 
-4. Deploy the operator in your namespace.
+{% include_relative install_kafka.md %}
```

Comment:
Should this be a prerequisite instead?
Maybe between lines 1&2 directly in this doc unless we're going to use it somewhere else.